addons/main/stringtable.xml

```diff
@@ -17,6 +18,7 @@
             <Chinesesimp>https://www.github.com/CBATeam/CBA_A3</Chinesesimp>
             <Portuguese>https://www.github.com/CBATeam/CBA_A3</Portuguese>
             <Russian>https://www.github.com/CBATeam/CBA_A3</Russian>
+            <French>https://www.github.com/CBATeam/CBA_A3</French>
```

Comment:
Not needed. If it's not there Arma defaults to the topmost one. And as all others are the same anyway..
Saves a tiny bit of disk space ;)

Reply:
But will be marked as missing in stringtable checker. Keep it.